libnetwork/controller_linux.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/docker/docker/libnetwork/osl"
 )
 
+// FirewallBackend returns the name of the firewall backend for "docker info".
+func (c *Controller) FirewallBackend() [][2]string {
+	if iptables.FirewalldRunning() {
```

Comment:
And pass the context here;
```suggestion
// FirewallBackend returns the name of the firewall backend for "docker info".
func (c *Controller) FirewallBackend(ctx context.Context) [][2]string {
	if iptables.FirewalldRunning(cox) {
```